app/assets/stylesheets/modules/_navigation.scss

```diff
@@ -9,21 +9,21 @@
 
 .nav a { color: #fefeff; }
 
-.navbar-bright {
+.navbar-right {
   background-color:#7f0f14;
   color:#fff;
   margin-bottom: 0;
   border-width: 0;
 }
 
-.navbar-bright a, .navbar-bright .navbar-brand, #masthead h4 {
+.navbar-right a, .navbar-right .navbar-brand, #masthead h4 {
   color:#fff;
   font-size:14px;
   text-transform: uppercase;
   letter-spacing:.08em;
 }
 
-.navbar-bright li > a:hover {
+.navbar-right li > a:hover {
```

Comment:
Merge rule `.navbar-right li > a:hover` with rule on line 12<br>Selector should have depth of applicability no greater than 2, but was 3